internal/uvm/start.go

```diff
@@ -115,6 +118,38 @@ func parseLogrus(vmid string) func(r io.Reader) {
 	}
 }
 
+// When using an external GCS connection it is necessary to send a ModifySettings request
+// for HvSockt so that the GCS can setup some registry keys that are required for running
+// containers inside the UVM. Usually this is done by the HCS immediately after the GCS
```

Comment:
maybe instead of saying "usually" here, say "in a non external GCS connection scenario"